conformance/lang/expressions/relational-expr/greater_than_expression.balt

```diff
@@ -0,0 +1,1162 @@
+   io:println(c > a); // @output false
+   io:println(a > c); // @output true
+   io:println(d > a); // @output false
+   io:println(b > d); // @output true
+   io:println(a > a); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression where one operand is negative int:Signed16 and other one is positive int:Signed16
+Labels: DecimalNumber, int:Signed16, relational-expr, unary-minus
+
+function init() {
+   int:Signed32 a = -1;
+   int:Signed32 b = 2;
+   int:Signed32 c = -40123;
+   int:Signed32 d = 2147483647;
+   io:println(a > b); // @output false
+   io:println(b > a); // @output true
+   io:println(d > c); // @output true
+   io:println(c > d); // @output false
+   io:println(a > d); // @output false
+   io:println(d > a); // @output true
+}
+
+Test-Case: output
+Description: Test greater than expression for user defined sub types of positive int
+Labels: DecimalNumber, module-type-defn, relational-expr, singleton-type, union-type
+
+type Ints -2|-1|0|1|2;
+
+function init() {
+   Ints a = 1;
+   Ints b = 2;
+   Ints c = 0;
+
+   io:println(a > b); // @output false
+   io:println(b > a); // @output true
+   io:println(a > c); // @output true
+   io:println(c > a); // @output false
+   io:println(b > c); // @output true
+   io:println(c > b); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression for user defined sub types of negative int
+Labels: DecimalNumber, module-type-defn, relational-expr, singleton-type, unary-minus, union-type
+
+type Ints -2|-1|-3|1|2;
+
+function init() {
+   Ints a = -1;
+   Ints b = -2;
+   Ints c = -3;
+
+   io:println(a > b); // @output true
+   io:println(b > a); // @output false
+   io:println(a > c); // @output true
+   io:println(c > a); // @output false
+   io:println(b > c); // @output true
+   io:println(c > b); // @output false
+}
+
+Test-Case: output
+Description: Test greater than expression for user defined sub types where one operand is positive int and other one is negative int
+Labels: DecimalNumber, module-type-defn, relational-expr, singleton-type, unary-minus, union-type
+
+type Ints -2|-1|0|1|2;
+
+function init() {
+   Ints a = 1;
+   Ints b = 2;
+   Ints c = -1;
+   Ints d = -2;
+
+   io:println(a > c); // @output true
+   io:println(c > a); // @output false
+   io:println(b > c); // @output true
+   io:println(c > b); // @output false
+   io:println(a > d); // @output true
+   io:println(d > a); // @output false
+   io:println(d > b); // @output false
+   io:println(b > d); // @output true
+}
+
+Test-Case: output
+Description: Test greater than expression for user defined sub types of integer built in sub types
+Labels: DecimalNumber, int:Signed8, int:Unsigned32, module-type-defn, relational-expr, union-type
+
+type Ints int:Signed8|int:Unsigned32;
+
+function init() {
+   Ints a = 12;
+   int:Signed8|int:Unsigned32 b = 13;
+
+   io:println(a > b); // @output false
+   io:println(b > a); // @output true
+}
+
```

Comment:
Can also test relational expressions between different subtypes of `int`.
For an example:
```ballerina
const ONE = 1;

type Ints int:Signed8|int:Unsigned32;

function init() {
   int:Signed16 a = 12;
   int:Signed8|int:Unsigned32 b = 13;

   io:println(a > b); 

   io:println(ONE > a);

   int:Unsigned16 c = 10;
   Ints d = 20;
   io:println(c > d);
}
```